moped-editor/src/components/forms/ControlledAutocomplete.js

```diff
@@ -0,0 +1,113 @@
+import Autocomplete from "@mui/material/Autocomplete";
+import TextField from "@mui/material/TextField";
+import { Controller } from "react-hook-form";
+
+/**
+ * Defines the callback handler for when a field value changes. This function keeps
+ * react-hook-form state in sync with the field input state. Since this is the default
+ * handler, it merely saves the exact value(s) selected into RFH state
```

Comment:
RHF state?

Reply:
react-hook-form: will update that! 🙏

Reply:
assumed it was a RFH/RHF typo! but its nice to have fewer acronyms